convert2rhel/main.py

```diff
@@ -90,7 +90,11 @@ def main():
 
         experimental_analysis = bool(os.getenv("CONVERT2RHEL_EXPERIMENTAL_ANALYSIS", None))
         loggerinst.task("Conversion analysis report")
-        report.summary(pre_conversion_results, include_all_reports=experimental_analysis)
+        report.summary(
+            pre_conversion_results,
+            with_colors=logger_module.should_disable_color_output(),
+            include_all_reports=experimental_analysis,
+        )
```

Comment:
If the previous suggestion is accepted, we could write this as 

```suggestion
        report.summary(
            pre_conversion_results,
            include_all_reports=experimental_analysis,
        )
```